src/widgets/radio.jsx

```diff
@@ -173,21 +288,15 @@ var Radio = React.createClass({
     render: function() {
         var choices = this.props.choices;
         var values = this.props.values || _.map(choices, () => false);
-        var revealNoneOfTheAbove = this._shouldRevealNoneOfTheAbove(choices,
-                                                                    values);
+
         choices = _.map(choices, (choice, i) => {
-            var content;
-            if (choice.isNoneOfTheAbove && !revealNoneOfTheAbove) {
-                content = "None of the above";
-            } else {
-                content = choice.content;
-            }
-            return {
-                // We need to make a copy, which _.pick does
-                content: this._renderRenderer(content),
+            return _.extend({}, {
```

Comment:
Since the object below is a new object, not an existing `this.props` or `this.props.something`, we don't need to use `_.extend` here. We usually use `_.extend` to clone existing objects and add changes to the new clone. Here we don't need to, because the whole object is new :).

So we can just write

    return {
        content: this._renderRenderer(choice.content),
        checked: values[i],
        correct: this.props.questionCompleted && values[i],
        clue: this._renderRenderer(choice.clue),
        isNoneOfTheAbove: choice.isNoneOfTheAbove
    };